src/components/Onfido/index.js

```diff
@@ -0,0 +1,70 @@
+import './index.css';
+import React from 'react';
+import * as OnfidoSDK from 'onfido-sdk-ui';
+import onfidoPropTypes from './onfidoPropTypes';
+import CONST from '../../CONST';
+
+class Onfido extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onfidoOut = undefined;
+    }
+
+    componentDidMount() {
+        this.onfidoOut = OnfidoSDK.init({
```

Comment:
Should we bind `this.onfidoOut` to `this`?

Reply:
No, I don't think that is necessary and it _could_ mess up the scope inside of `this.onfidoOut.tearDown()` if there is a reference to `this`.